packages/arb-token-bridge-ui/src/token-bridge-sdk/EthDepositStarter.ts

```diff
@@ -85,35 +85,65 @@ export class EthDepositStarter extends BridgeTransferStarter {
     // no-op
   }
 
-  public async transferEstimateGas({ amount, signer }: TransferEstimateGas) {
+  public async transferEstimateGas({
+    amount,
+    signer,
+    destinationAddress
+  }: TransferEstimateGas) {
     const address = await getAddressFromSigner(signer)
 
     return depositEthEstimateGas({
       amount,
       address,
       parentChainProvider: this.sourceChainProvider,
-      childChainProvider: this.destinationChainProvider
+      childChainProvider: this.destinationChainProvider,
+      destinationAddress
     })
   }
 
-  public async transfer({ amount, signer }: TransferProps) {
+  public async transfer({ amount, signer, destinationAddress }: TransferProps) {
     const address = await getAddressFromSigner(signer)
     const ethBridger = await this.getBridger()
 
-    const depositRequest = await ethBridger.getDepositRequest({
-      amount,
-      from: address
-    })
+    const depositRequest = destinationAddress
```

Comment:
Similar check here - we can define a variable like 

`const isCustomDestinationTransfer = destinationAddress && (destinationAddress !== address)` 

and use that to be sure that we only use the custom destination flow when this condition arises?

Reply:
Should also lowercase both just to be sure